chpl/chpl-service/src/main/java/gov/healthit/chpl/permissions/domains/activity/GetPendingListingMetadataActionPermissions.java

```diff
@@ -0,0 +1,20 @@
+package gov.healthit.chpl.permissions.domains.activity;
+
+import org.springframework.stereotype.Component;
+
+import gov.healthit.chpl.permissions.domains.ActionPermissions;
+
+@Component("activityGetPendingListingMetadataActionPermissions")
+public class GetPendingListingMetadataActionPermissions extends ActionPermissions {
+
+    @Override
+    public boolean hasAccess() {
+        return getResourcePermissions().isUserRoleAdmin() || getResourcePermissions().isUserRoleOnc();
```

Comment:
How does this work with the rule of an ACB should be able to see pending listing activity belonging to their ACB?

Reply:
Currently, there is no other report other than the User Actions Activity report that uses this data.  I do not believe that there is any current functionality to get pending listing activity.  Since it is a new endpoint, I feel it only needs security limiting it to users who can access the User Actions Activivty report.